### PR TITLE
LRDOCS-8278 Ratings Wordsmithed

### PR DESCRIPTION
**Pull request by sez11a**
https://issues.liferay.com/browse/LRDOCS-8278

This was originally an LPS, but there was also a closed ticket for it (see above). I reopened the ticket; I think this article supersedes the one that somebody else ported, and maybe we can remove the other one. 

@Alec-Shay

### CODE REVIEW

**Comment by ambrinchaudhary** [comment]
LGTM! 
There are things on the [Technical Documentation Draft](https://docs.google.com/document/d/1xFdMzQunwubGlXMN1bDSliZ21pq-8Er0D6PjaVhK7jE/edit) that are not covered here, but as I can see in the first commit's comment, I gues that information will be updated somewhere else, is that right?

**Comment by Alec-Shay** [comment]
Thanks @ambrinchaudhary!

And that's right, I [commented on the LPS](https://issues.liferay.com/browse/LPS-120400?focusedCommentId=2271041&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-2271041) but I'll also update the other subtask tickets to better describe what we intend. For context, we may be frequently reorganizing content like this because we want to try to present each article with a single distinct topic as much as possible (and in cases like this, we can split it into user-side articles and developer-side documentation).

Basically, the technical documentation on the taglib usage will be presented in separate articles (one tutorial demonstrating how to generally enable ratings for a custom application, and one showing how to implement a `RatingsDataTransformer` with it). Very soon you should also get a draft for the first of those other tutorials for review, too, so soon the only thing missing will be the `RatingsDataTransformer` tutorial.

**Comment by jrhoun** [comment]
Merged. Thanks.
